Makefile

```diff
@@ -29,11 +29,12 @@ inertia-tagged:
 
 # Remove Inertia binaries
 clean:
-	rm -f ./inertia 
+	rm -f ./inertia
 	find . -type f -name inertia.\* -exec rm {} \;
 
 lint:
 	PATH=$(PATH):./bin bash -c './bin/gometalinter --vendor --deadline=60s ./...'
+	cd ./daemon/web; npm run lint
```

Comment:
Hard-cd-ing into `./daemon/web` might have weird consequences if more commands are added, etc.

Might want to use something similar to the [other web commands](https://github.com/ubclaunchpad/inertia/blob/b92709169e4f29d091a29653f4b1d229d1756d39/Makefile#L105):

```bash
web-run:
   (cd ./daemon/web; npm start)
```

This will cd into the directory only for that particular command

Reply:
Haha, no worries, feel free to add in as many comments as you want.

That makes a lot more sense! Will make the change.